pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [project]
 name = "jdaviz"
 description = "Astronomical data analysis development leveraging the Jupyter platform"
-requires-python = ">=3.9"
+requires-python = ">=3.10"
```

Comment:
Only this line seems relevant. Is the rest some overzealous ruff gone wild?

Reply:
yes that looks like the case; I'll prune the other changes